package.json

```diff
@@ -39,7 +39,8 @@
     "version-check": "npm run sync-external && node version-check.js",
     "build-external": "node ./external/build-external.js",
     "maybe-sync-external": "node ./external/maybe-sync-external.js",
-    "sync-external": "node ./external/sync-external.js"
+    "sync-external": "node ./external/sync-external.js",
+    "check-links": "node ./gulp-tasks/check/links.js"
```

Comment:
Rather export the task from the main gulpfile, so one is able to just run `npx gulp checkLinks`. In an effort to reduce the number of NPM scripts.